sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -59,6 +59,9 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <property name="onCommentFormat" value="CHECKSTYLE.ON\: ([\w\|]+)"/>
     <property name="checkFormat" value="$1"/>
   </module>
+  <module name="SuppressionFilter">
```

Comment:
No, the file location in the pom gets passed in here.   Eclipse/m2e/checkstyle uses the file location in the pom to copy to target and then sets a property which this then picks up.